docs/usage/permissioning.md

```diff
@@ -0,0 +1,287 @@
+# Permissioning
+
+Dragonchain has a built-in permissioning system, allowing specific api keys to
+be either allowed or denied for certain dragonchain actions via the Dragonchain
+api.
+
+In order to use these permissions, policy documents are given to api keys which
+determine which actions are allowed and forbidden.
+
+## Setting Permissions
+
+In order to set permissions, create or update an api key with a permissions
+document (schema explained below).
+
+By default, an api key with no explicitly set permissions will have access to
+every endpoint, except create/update/delete api keys.
+
+Keep in mind that the ability to create or update api keys is effectively a
+root-level permission, because keys can be created or modified with elevated
+permissions via those endpoints.
+
+### Root API Key
+
+The root api key (in the kubernetes secret when deploying the chain) will
+always have permission to use any Dragonchain api endpoint (aside from
+endpoints for dragon net interchain actions, which are reserved for
+dragonchain-to-dragonchain communication).
+
+The root api key cannot be deleted.
+
+## Permissions Document
+
+The permissions document is how permissions are defined per api key.
+
+It is a JSON object containing the document version (currently only `"1"`), a
```

Comment:
Should be same paragraph as above, or shouldn't start with `it`. Adds ambiguity.